examples/withEmbeddedDatabase/solutions/angularNestJS/README.md

```diff
@@ -0,0 +1,32 @@
+# Solution for Web systems with embedded database using Angular and Nest.js
+
+1. To get install, you need install Nest CLI using ```npm i -g @nestjs/cli```
+2. After, using ```nest new backend``` to  create a new project directory, [this folder](backend)
```

Comment:
Explain Backend implementation. What layers do you need to create and what files in each layer?